src/components/token/TokenMarkers.js

```diff
@@ -25,16 +25,26 @@ const TokenMarkers = (props) => {
   }
 
   const bannedMarker = () => {
-    if (!token.banned) {
+    if (!token.meta || !token.meta.banned) {
       return null;
     }
+    let banIcon = (
+      <i className="fa fa-exclamation-triangle fa-lg text-danger" title="Banned"></i>
+    )
+    if (token.meta && token.meta.reason) {
```

Comment:
Here you already know that you have `token.meta` because of the `if` above, right?

Reply:
Yes, removing the unnecessary check